Automattic/studio
# Update Assistant-related icons

Improves Assistant-related icons to match designs. 

## Proposed Changes

- Removes static `<ReturnIcon />` and `<MenuIcon />` in favor of `@wordpress/icons`: `keyboardReturn` and `moreVertical`.
- Updates `<AssistantIcon />` to regular function component ([#](https://github.com/Automattic/studio/pull/202#discussion_r1626074876)) 

## Testing Instructions

1. Start app with AI Assistant flag enabled: `STUDIO_AI=true npm start`
2. Navigate to Assistant tab
3. **Type text** in the `<AIInput />` component
4. Observe the following icons (left to right -- `Assistant`, `keyboardReturn`, and `moreVertical`):

<img width="636" alt="Icons" src="https://github.com/Automattic/studio/assets/643285/fb6b4fb7-ee32-4dc2-bb68-7217c26ba757">


## Pre-merge Checklist
